showy.typ

```diff
@@ -55,11 +55,40 @@
     width: 1pt,
     dash: "solid"
   ),
+  shadow: none,
   title: "",
   breakable: false,
   ..body
 ) = {
-  block(
+  /*
+   * Optionally create a wrapper
+   * function to add a shadow.
+   */
+  let wrap = (sbox) => sbox
+  if shadow != none {
+    if type(shadow.at("offset", default: 4pt)) != "dictionary" {
+      shadow.offset = (
+        x: shadow.at("offset", default: 4pt),
+        y: shadow.at("offset", default: 4pt)
+      )
+    }
+    wrap = (sbox) => move(
+      dx: shadow.offset.x,
+      dy: shadow.offset.y,
+      rect(
+        radius: frame.at("radius", default: 5pt),
+        fill:   shadow.at("color", default: luma(128)),
+        outset: 0pt,
+        inset:  0pt,
+        move(
+          dx:-1 * shadow.offset.x,
+          dy:-1 * shadow.offset.y,
+          sbox
+        )
+      )
+    )
+  }
+  wrap(block(
```

Comment:
The `wrap()` function here is not closed. The parenthesis should be in the penultimate line of code in this file. I can not change it from here, so you should make a new commit changing it ;)